src/overlays/actors/ovl_kaleido_scope/z_kaleido_scope.c

```diff
@@ -76,7 +79,54 @@
 
 #pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_kaleido_scope/func_80825C14.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_kaleido_scope/KaleidoScope_Draw.s")
+extern const char D_8082FD78[], D_8082FD94[];
+
+void KaleidoScope_Draw(GlobalContext* globalCtx) {
+    Input* input = &globalCtx->state.input[0];
+    PauseContext* pCtx = &globalCtx->pauseCtx;
+    InterfaceContext* iCtx = &globalCtx->interfaceCtx;
```

Comment:
if these temps are necessary, I would go with the full name of the context to make it easier to read

Reply:
i didnt mean to highlight input there.
pCtx -> pauseCtx
iCtx -> interfaceCtx